src/main/java/edu/hawaii/its/api/wrapper/GetMembersResults.java

```diff
@@ -43,4 +47,58 @@ public List<GetMembersResult> getMembersResults() {
         }
         return getMembersResults;
     }
+
+    public void removeMember(String groupPath, String uhIdentifier) {
+        WsGetMembersResults wsGetMembersResults = this.wsGetMembersResults;
+        WsGetMembersResult[] wsGetMembers = wsGetMembersResults.getResults();
+        WsGetMembersResult[] updatedWsGetMembersResults = Arrays.stream(wsGetMembers)
+                .map(wsGetMembersResult -> {
+                    if (wsGetMembersResult.getWsGroup() != null && groupPath.equals(wsGetMembersResult.getWsGroup().getName())) {
+                        WsGetMembersResult filteredResult = new WsGetMembersResult();
+                        filteredResult.setWsGroup(wsGetMembersResult.getWsGroup());
+                        filteredResult.setResultMetadata(wsGetMembersResult.getResultMetadata());
+                        WsSubject[] filteredSubjects = Arrays.stream(wsGetMembersResult.getWsSubjects())
+                                .filter(wsSubject -> !uhIdentifier.equals(wsSubject.getId()))
+                                .toArray(WsSubject[]::new);
+
+                        filteredResult.setWsSubjects(filteredSubjects);
+                        return filteredResult;
+                    } else {
+                        return wsGetMembersResult;
+                    }
+                })
+                .toArray(WsGetMembersResult[]::new);
+        wsGetMembersResults.setResults(updatedWsGetMembersResults);
+    }
+
+    public void removeMembers(String groupPath, List<String> uhIdentifiers) {
+        WsGetMembersResults wsGetMembersResults = this.wsGetMembersResults;
+        WsGetMembersResult[] wsGetMembers = wsGetMembersResults.getResults();
+        WsGetMembersResult[] updatedWsGetMembersResults = Arrays.stream(wsGetMembers)
+                .map(wsGetMembersResult -> {
+                    if (wsGetMembersResult.getWsGroup() != null && groupPath.equals(wsGetMembersResult.getWsGroup().getName())) {
+                        WsGetMembersResult filteredResult = new WsGetMembersResult();
+                        filteredResult.setWsGroup(wsGetMembersResult.getWsGroup());
+                        filteredResult.setResultMetadata(wsGetMembersResult.getResultMetadata());
+                        WsSubject[] filteredSubjects = Arrays.stream(wsGetMembersResult.getWsSubjects())
+                                .filter(wsSubject -> !uhIdentifiers.contains(wsSubject.getId()))
+                                .toArray(WsSubject[]::new);
+
+                        filteredResult.setWsSubjects(filteredSubjects);
+                        return filteredResult;
+                    } else {
+                        return wsGetMembersResult;
+                    }
+                })
+                .toArray(WsGetMembersResult[]::new);
+        wsGetMembersResults.setResults(updatedWsGetMembersResults);
+    }
+
+    @JsonIgnore
+    public GetMembersResult getMembersByGroupPath(String groupPath){
```

Comment:
Correct me if I am wrong, the `@JsonIgnore` is not needed now.

Reply:
I just added the annotation for avoiding the different api spec. I checked api data without @JsonIgnore, and it would be okay to remove it.